src/ex1/scripts/domScripts.js

```diff
@@ -0,0 +1,195 @@
+const taskInput = document.querySelector(".task-input");
+const addTaskButton = document.querySelector(".task-add-btn");
+const tasksList = document.querySelector(".tasks-list");
+const filterOptiopn = document.querySelector(".filter-tasks");
+const searchInput = document.querySelector(".search-input");
```

Comment:
querySelector might return an array of elements.
for unique elements such as buttons - add, clear, search, etc. you should use ids and not classes to identify the elements and then use `getElementById` instead of `querySelector`